src/ui/osx/TogglDesktop/MainWindowController.m

```diff
@@ -215,6 +215,9 @@ - (void)displayOnlineState:(NSNumber *)status
 		case 2 :
 			[[SystemMessage shared] presentOffline:@"Error" subTitle:@"Offline, Toggl not responding"];
 			break;
+        case 3 :
+            [[SystemMessage shared] presentOffline:@"Error" subTitle:@"Too many requests, sync delayed by 1 minute"];
+            break;
```

Comment:
I suggest that we could treat the RateLimit as a normal error message and present in DisplayError notification as usual.

https://github.com/toggl/toggldesktop/pull/3388 could be easier and less error prone